internal/control_loop/direct.go

```diff
@@ -31,27 +31,19 @@ func NewDirectControlLoop(
 func (l *DirectControlLoop) Cycle(target int, current int) int {
 	loopTime := time.Now()
 
-	dt := loopTime.Sub(l.lastTime).Seconds()
+	//dt := loopTime.Sub(l.lastTime).Seconds()
 
 	l.lastTime = loopTime
 
 	var stepTarget = float64(target)
 	if l.maxPwmChangePerCycle != nil {
-		// the pwm adjustment depends on the direction and
-		// the time-based change speed limit.
-		stepTarget = float64(*l.maxPwmChangePerCycle) * dt
+		maxChangeValue := *l.maxPwmChangePerCycle
+		// TODO: figure out how to normalize the change to the timedelta
+		//stepTarget = float64(*l.maxPwmChangePerCycle) * dt
```

Comment:
I have removed this code for now, and I am not sure if we need it.
The PID Control Loop requires the timedelta between cycles to work properly, but I am not so sure about the Direct Control Loop. It could just as well be a feature of this type of Control Loop that it does **not** account for variations in timing.

@jwefers What is your use-case for this? Do you think its fine as is or do we need to think about the "passed time since the last cycle" in a "direct" control loop?